### ISSUE TITLE
Add interface animations/transitions 

### ISSUE THREAD

**Issue by ruyadorno** [yeoman/yeoman-app#12]
Although the styling is complete the app still lacks animations, those are essential for Material Design.

**Comment by stefanbuck** [comment]
@ruyadorno is this issue still valid? The current master looks pretty smooth to me!

**Comment by ruyadorno** [comment]
yeah, although **material-ui** really improved a lot the quality of animations in their components I still have some ideas in mind for the app interface, such as transitions when changing question prompts, smoother transition from clicking on a generator and opening the install process, etc
